packages/story-editor/src/components/floatingMenu/elements/borderWidthAndColor.js

```diff
@@ -137,8 +137,10 @@ function BorderWidthAndColor() {
           value={border.color || BLACK}
           onChange={handleColorChange}
           hasInputs={false}
-          hasEyeDropper={false}
+          hasEyedropper={false}
           allowsOpacity={canHaveBorderOpacity}
+          pickerHasEyedropper={false} // override shared floating menu Color component TODO https://github.com/GoogleForCreators/web-stories-wp/issues/11024
+          allowsSavedColors={false} // if this changes to true, need to change allowsGradient to false because borders cannot be gradient.
```

Comment:
💭 if one of `allowGradient` or `allowSavedColors` must be false, then we should add that in here:
```suggestion
          pickerHasEyedropper={false} // override shared floating menu Color component TODO https://github.com/GoogleForCreators/web-stories-wp/issues/11024
          // Borders cannot be gradients.
          // One of `allowSavedColors` or `allowGradient` must be false.
          allowsSavedColors={!allowGradient} 
          allowGradient={allowGradient}
```

Reply:
Also - can we add a small unit test here?